src/Dapr.Actors/Runtime/ActorManager.cs

```diff
@@ -129,7 +133,7 @@ async Task<object> RequestFunc(Actor actor, CancellationToken ct)
                 else
                 {
                     var errorMsg = $"Method {string.Concat(methodInfo.DeclaringType.Name, ".", methodInfo.Name)} has more than one parameter and can't be invoked through http";
-                    ActorTrace.Instance.WriteError(TraceType, errorMsg);
+                    this.logger.LogError(errorMsg);
```

Comment:
In general we shouldn't log where exceptions are **thrown** - it ends up being redundant because exceptions end up being logged where they are handled.